Description: Upstream changes introduced in version 2:7.2.445+hg~cb94c42c0e1a-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 vim (2:7.2.445+hg~cb94c42c0e1a-1) unstable; urgency=low
 .
   * debian/control:
     + Use Breaks instead of Conflicts for the gvimtutor move.
     + Use linux-any wildcards for linux-specific Build-Depends.  (Closes:
       #587866)
     + Bump Standards-Version to 3.9.0.0.
   * Pull patches 439 - 445.
   * runtime/ftplugin/debchangelog.vim: Use “silent!” to avoid errors when
     disabling the menu.  Thanks to Jan Christoph Ebersbach for the patch.
 .
 The person named in the Author field signed this changelog entry.
Author: James Vega <jamessan@debian.org>
Bug-Debian: http://bugs.debian.org/587866

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- vim-7.2.445+hg~cb94c42c0e1a.orig/runtime/ftplugin/debchangelog.vim
+++ vim-7.2.445+hg~cb94c42c0e1a/runtime/ftplugin/debchangelog.vim
@@ -3,9 +3,9 @@
 " Maintainer:   Debian Vim Maintainers <pkg-vim-maintainers@lists.alioth.debian.org>
 " Former Maintainers:   Michael Piefel <piefel@informatik.hu-berlin.de>
 "                       Stefano Zacchiroli <zack@debian.org>
-" Last Change:  2010-04-29
+" Last Change:  2010-07-11
 " License:      GNU GPL, version 2.0 or later
-" URL:          http://hg.debian.org/hg/pkg-vim/vim/raw-file/tip/runtime/ftplugin/debchangelog.vim
+" URL:          http://hg.debian.org/hg/pkg-vim/vim/file/unstable/runtime/ftplugin/debchangelog.vim
 
 " Bug completion requires apt-listbugs installed for Debian packages or
 " python-launchpadlib installed for Ubuntu packages
@@ -228,7 +228,7 @@ endfunction
 
 augroup changelogMenu
 au BufEnter * if &filetype == "debchangelog" | call <SID>MakeMenu() | endif
-au BufLeave * if &filetype == "debchangelog" | aunmenu Changelog | endif
+au BufLeave * if &filetype == "debchangelog" | silent! aunmenu Changelog | endif
 augroup END
 
 " }}}
